### PR TITLE
Patch appear of more arrow on tabs

### PR DESCRIPTION
**Pull request by DOMEYD**
When window is resized, the tabsSuperPanel (container) width was not updated and cause not displaying more arrow.

### CODE REVIEW

File: static/admin/novius-os/js/jquery.novius-os.ostabs.js [hunk 1]
```diff
@@ -56,6 +56,13 @@ define('jquery-nos-ostabs',
                         self._firePanelEvent(self.panels.eq(o.selected), $.Event('resizePanel'));
                     }
                 });
+		self._resize();
```
Comment (Meroje) [inline static/admin/novius-os/js/jquery.novius-os.ostabs.js]: Shouldn't this be one line higher in the resize callback ?

**Comment by Meroje** [comment]
:+1:

File: static/admin/novius-os/js/jquery.novius-os.ostabs.js [hunk 2]
```diff
@@ -55,9 +55,16 @@ define('jquery-nos-ostabs',
                     if (o.selected !== null) {
                         self._firePanelEvent(self.panels.eq(o.selected), $.Event('resizePanel'));
                     }
+		    self._resize();
```
Comment (jguyomard) [inline static/admin/novius-os/js/jquery.novius-os.ostabs.js]: Don't use tab to indent

**Comment by jguyomard** [comment]
You had to rebuild `nos.min.js` bundle (since your last commit)

**Pull request merged**